src/Expression.php

```diff
@@ -630,7 +630,7 @@ private function getCastValue($v): ?string
      *
      * @return string[][]|null[][]
      */
-    public function get(): array
+    public function getRows(): array
```

Comment:
We need to have the old method issuing deprecated warning for 1 version iteration